pages/test-pickup-times/index.tsx

```diff
@@ -0,0 +1,416 @@
+import Head from "next/head"
+import { useState, useRef, useEffect } from "react"
+import { useRouter } from "next/router"
+import { parse, stringify } from "qs"
+import {
+  Heading,
+  Form,
+  FormField,
+  FormRow,
+  TextInput,
+  Select,
+  HorizontalRule,
+  ButtonGroup,
+  Button,
+} from "@nypl/design-system-react-components"
+import type { TextInputRefType } from "@nypl/design-system-react-components"
+
+import Layout from "../../src/components/Layout/Layout"
+
+import { BASE_URL, SITE_NAME } from "../../src/config/constants"
+import {
+  getPickupTimeEstimate,
+  overrideNow,
+} from "../../src/utils/pickupTimeEstimator"
+
+// Prepare fulfillment dropdown values:
```

Comment:
Small nit but I think it would be good to move most of the stuff preceding the component to a separate pickupTimesUtils file.

Reply:
I moved most of the util-like things into `src/utils/testPickupTimesUtils.ts`